src/moodleapi/index.ts

```diff
@@ -0,0 +1,3 @@
+/* tslint:disable */
+/* eslint-disable */
```

Comment:
I don't think we actually need these. Removing them will also surface a linter issue for this file not ending with a newline.